hi-ml/src/health_ml/runner.py

```diff
@@ -0,0 +1,318 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+import argparse
+import logging
+import os
+import param
+import sys
+import uuid
+from pathlib import Path
+from typing import Any, Dict, Optional, Tuple
+
+import matplotlib
+
+# Add hi-ml packages to sys.path so that AML can find them
+himl_root = Path(__file__).parent.parent.parent.parent
+print(f"Starting the himl runner at {himl_root}")
+print(f"health_ml pkg: {himl_root}")
+health_ml_pkg = himl_root / "hi-ml" / "src"
+health_azure_pkg = himl_root / "hi-ml-azure" / "src"
```

Comment:
do we still need those, now that you are no longer changing working directory?

Reply:
I think so - otherwise in AML the statements underneath to import from hi-ml would fail? (using hi-ml as a submodule here)